contracts/cwd-core/src/contract.rs

```diff
@@ -78,13 +77,9 @@ pub fn instantiate(
         ITEMS.save(deps.storage, key, &value)?;
     }
 
-    // Save total and active proposal module counts
     TOTAL_PROPOSAL_MODULE_COUNT.save(deps.storage, &0)?;
     ACTIVE_PROPOSAL_MODULE_COUNT.save(deps.storage, &0)?;
 
-    // Set created timestamp.
-    CREATED_TIMESTAMP.save(deps.storage, &env.block.time)?;
```

Comment:
**praise**: sorry i missed this one from my other PR

Reply:
np! I missed it too.